src/constants/jsonRpcEndpoints.ts

```diff
@@ -3,7 +3,16 @@ import { SupportedChainId } from './chains'
 /**
  * Fallback JSON RPC endpoints if integrator does not provide one
  */
-export const JSON_RPC_FALLBACK_ENDPOINTS: { [key in SupportedChainId]?: string[] } = {
+export const JSON_RPC_FALLBACK_ENDPOINTS: { [chainId: number]: string[] } = {
   [SupportedChainId.MAINNET]: ['https://cloudflare-eth.com'],
-  [SupportedChainId.RINKEBY]: ['https://cloudflare-eth.com/v1/rinkeby'],
+  [SupportedChainId.ROPSTEN]: ['https://rpc.ankr.com/eth_ropsten'],
+  [SupportedChainId.RINKEBY]: ['https://rpc.ankr.com/eth_rinkeby'],
+  [SupportedChainId.GOERLI]: ['https://rpc.ankr.com/eth_goerli'],
```

Comment:
This is my first time hearing of ankr--what's the summary on this change? (Mostly interested in the choice for Optimism.)

Reply:
just adding more free, public fallback RPC URLs for all the chains we support  -- will note in docs that it's *highly* recommended for integrators to provide their own, esp in prod environments

cc @akarys92